wordpress-mobile/WordPress-Android
# WP Stories integration - Media Picker part 1: device media

This builds on top of #11931 

This PR implements the connections necessary for the WP Stories Activity to interact with the host app (WPAndroid) media picker and be fed the media selection properly.

This PR only works for device media, replicating the existing functionality in the portkey demo app on the WordPress Android app.  The WPMedia and Stock photos sources will be added in subsequent PRs.

To test:
1. run the app
2. select a site
3. tap the FAB button on the "Me" screen
4. select Story
5. observe the Story creation experience appears (camera preview)
6. tap on the upload icon to the left of the camera shutter
7. observe the WPAndroid media picker gets launched
8. select some media items and tap on the checkmark on the top toolbar to indicate you're done selecting
9. observe these media items are added as background for as many Story pages as media items were selected.

Also test:
1. tapping on the camera FAB in the media picker should bring you back to the Stories capture mode.

PR submission checklist:
